setup.py

```diff
@@ -1,7 +1,5 @@
 """Setup file for building/installing flake8-sfs."""
 
-from __future__ import with_statement
```

Comment:
Not useless, it was important to ensure ``python2 setup.py ...`` works as well as possible. i.e. Clear error message, not an ugly exception.

Reply:
Well, if you require python >= 3.7 and think that the exception is ugly - there should be a bug somewhere else.

Reply:
Less and less important, I grant you - but that's why the future-import was there ;)